docs/microphone.rst

```diff
@@ -0,0 +1,114 @@
+Microphone **V2**
+*****************
+
+.. py:module:: microbit.microphone
+
+This object lets you access the built-in microphone available on the
+micro:bit **V2**. It can be used to respond to sound. The microphone input
+is located on the front of the board alongside a microphone activity LED,
+which is lit when the microphone is in use.
+
+.. image:: microphone.png
+    :width: 300px
+    :align: center
+    :height: 240px
+    :alt: micro:bit with microphone LED on
+
+Sound events
+============
+The microphone can respond to a pre-defined set of sound events that are
+based on the amplitude and wavelength of the sound.
+
+.. py:class:: SoundEvent  
+
+    Value to represent loud sound events, like clapping or shouting
+    ``SoundEvent.LOUD`` = ``SoundEvent.LOUD('loud')``.
+
+    Value to represent quiet sound events, like speaking or background music
+    ``SoundEvent.QUIET`` = ``SoundEvent.QUIET('quiet')``.
```

Comment:
I'm not sure this very clear what its:

![image](https://user-images.githubusercontent.com/29712657/100781015-642a3a00-3402-11eb-85cb-8cfb6ebc9fe4.png)


It's a class, but we are not listing any methods or properties, and there is nothing for the user to do with it. The description explain a bit what to do with it, but the description is not clear what "value" means in this context. 

I think it might be better to convert this into a paragraph that describes that this class have instances already created and that can be used from the microphone methods.

Reply:
Done